desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/OverflowingItem.tsx

```diff
@@ -67,15 +60,11 @@ const OverflowingItem: React.FC<OverflowingItemProps> = ({
     <OverflowTooltip isOverflowing={isOverflown} title={label} toolTipTriggers={['hover', 'focus']}>
       <Button
         ref={textElementRef}
-        className={classNames(
-          `hue-path-browser__overflowing-label`,
-          `hue-path-browser__overflowing-${componentType}`,
-          isLastBreadcrumb ? `hue-path-browser__overflowing-last-breadcrumb` : ``
-        )}
+        className="hue-path-browser__overflowing-label"
         onClick={() => {
-          handleFilePathChange(url);
+          onFilepathChange(url);
```

Comment:
The only thing the url string does is to flow in via a prop and then back as an argument in the callback function. So we can skip the whole url part and let the parent component deal with that in its onClickHandler.